cmake/Templates/GoogleTest.CMakeLists.txt.cmake

```diff
@@ -5,9 +5,9 @@ include(ExternalProject)
 
 ExternalProject_Add(GoogleTest
                     GIT_REPOSITORY    https://github.com/google/googletest.git
-                    GIT_TAG           release-1.8.0
+                    GIT_TAG           release-1.10.0
```

Comment:
Note I bumped this since we've been using this version via conda anyway: https://github.com/rapidsai/integration/blob/branch-0.16/conda/recipes/versions.yaml#L74-L75

Reply:
Well it looks like our code didn't like that, time to revert it for the time being...